src/black_primer/primer.json

```
@@ -95,7 +95,7 @@
     },
     "pandas": {
       "cli_arguments": ["--experimental-string-processing"],
-      "expect_formatting_changes": true,
+      "expect_formatting_changes": false,
```

```suggestion
      "expect_formatting_changes": true,
```
And now it needs changes again? Compare https://github.com/psf/black/runs/4446024299?check_suite_focus=true